sonar-db/src/main/resources/org/sonar/db/component/SnapshotMapper.xml

```diff
@@ -195,11 +195,15 @@
     period1_mode, period2_mode, period3_mode, period4_mode, period5_mode,
     period1_param, period2_param, period3_param, period4_param, period5_param,
     period1_date, period2_date, period3_date, period4_date, period5_date)
-    values (#{uuid, jdbcType=VARCHAR}, #{parentId}, #{rootId}, #{rootComponentUuid, jdbcType=VARCHAR}, #{componentUuid, jdbcType=VARCHAR}, #{createdAt}, #{buildDate}, #{status, jdbcType=VARCHAR},
-    #{purgeStatus, jdbcType=VARCHAR}, #{last}, #{scope, jdbcType=VARCHAR}, #{qualifier, jdbcType=VARCHAR}, #{version, jdbcType=VARCHAR}, #{path, jdbcType=VARCHAR}, #{depth},
+    values (#{uuid, jdbcType=VARCHAR}, #{parentId, jdbcType=BIGINT}, #{rootId, jdbcType=BIGINT},
+    #{rootComponentUuid, jdbcType=VARCHAR}, #{componentUuid, jdbcType=VARCHAR}, #{createdAt, jdbcType=BIGINT},
+    #{buildDate, jdbcType=BIGINT}, #{status, jdbcType=VARCHAR},
+    #{purgeStatus, jdbcType=INTEGER}, #{last}, #{scope, jdbcType=VARCHAR}, #{qualifier, jdbcType=VARCHAR},
+    #{version, jdbcType=VARCHAR}, #{path, jdbcType=VARCHAR}, #{depth},
     #{period1Mode, jdbcType=VARCHAR}, #{period2Mode, jdbcType=VARCHAR}, #{period3Mode, jdbcType=VARCHAR}, #{period4Mode, jdbcType=VARCHAR}, #{period5Mode, jdbcType=VARCHAR},
     #{period1Param, jdbcType=VARCHAR}, #{period2Param, jdbcType=VARCHAR}, #{period3Param, jdbcType=VARCHAR}, #{period4Param, jdbcType=VARCHAR}, #{period5Param, jdbcType=VARCHAR},
-    #{period1Date}, #{period2Date}, #{period3Date}, #{period4Date}, #{period5Date})
+    #{period1Date, jdbcType=BIGINT}, #{period2Date, jdbcType=BIGINT}, #{period3Date, jdbcType=BIGINT},
+    #{period4Date, jdbcType=BIGINT}, #{period5Date, jdbcType=BIGINT})
```

Comment:
shouldn't we have one row per line to make merges and diffs more convenient (as we would like to have for DB unit files)?